src/bin/find.js

```diff
@@ -38,3 +56,9 @@ Object.keys(options).forEach(function findRules(option) {
 if (processExitCode) {
   process.exit(processExitCode)
 }
+
+function getOutputRuleCellMapper(width) {
```

Comment:
If possible it will be great to see it along with this PR. Other wise just create an issue for cleaning up dups.